app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] Kindly make sure that your comments controller has `new` and `create` actions. Currently, you have the `create` action but you are missing the `new` action. The purpose of a new action in a Rails controller is to display a form for creating a new instance of a model. So make sure that you add the `new` action to display a new form for the comment model. :pray: